### PR TITLE
Refactor component structure

### PR DESCRIPTION
**Pull request by Keelando**
separated page sections into their own components as a partial refactor

### CODE REVIEW

File: src/components/CellViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react"
+import { PatchType, Orientation, EdgeType, ActivityType } from "../slices"
+import { VisualArrayCell } from "../slices"
+// import CompilationSwitch from "./CompilationSwitch"
```
Comment (gwwatkin) [inline src/components/CellViewer.tsx:5]:
```suggestion
```

**Review by gwwatkin — APPROVED** [review]
Glad to see the components in a more organized fashion.

Just one thing is that when lots of code is moved around the diffs are hard to follow and if there were changes I probably missed them. A way to avoid that is to do on code move -> one commit

**Comment by Keelando** [comment]
I concur about the moves, will keep that in mind. I found the process of refactoring to be very non-linear and with many dead-ends. It's far from perfect, and will probably pick away at it as we go along.

I didn't make any functional changes to anything other than separating code into new files and namespacing.

**Review by alexnguyenn — APPROVED** [review]
👍
